### DISCUSSION TITLE
Column value

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6301] [Help] [answered]
Hey everyone.
I want to get the column value of the last row in a  specific table.
Can someone help me to do this?

**Comment by melohagan** [comment, marked as answer]
Hey @rafikApp 

What you could do is sort a data provider by Created By ascending or Auto ID ascending. Then you could grab the first row of the data provider - which would be your last in your table.

E.g. `{{ Data Provider.Rows.0.ColumnName }}`

**Comment by rafikApp** [comment]
Thank you it works.
